Backpack/StarRating/Classes/BPKStarRating.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "BPKStar.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(StarRating) @interface BPKStarRating : UIView
+
+/**
+ * Rating value the star rating view will displaying by rounding the value to full and half stars between 0.0 and 5.0.
+ * Example: setting 4.3 to rating will render 5 stars where the first 4 of them are filled with
```

Comment:
So does it always round down? I would expect a value of 4.3 to round to 4.5...

Reply:
Although maybe it's more traveller-first to always round down...

Reply:
It does round to 4.5, but the docs string could be improved to make it clearer

Reply:
Good point, made the example in the doc clearer